App/VSTS/CopyButton.tsx

```diff
@@ -0,0 +1,35 @@
+/// <reference path="../../office.d.ts" />
+import * as React from 'react';
+
+
+interface ICopyButtonProps {
+  workItemHyperlink: string;
+}
+
+export class CopyButton extends React.Component<ICopyButtonProps, {}> {
+
+  public render(): React.ReactElement<{}> {
+    console.log('got to copybutton');
+    document.addEventListener('copy', this.setClipboardHandler);
+
+    return (
+      <div>
+      <button className='ms-Button' onClick={this.handleClick}>
+      <span className='ms-Icon ms-Icon--copy'></span>
+      {'   '}Copy to Clipboard
```

Comment:
Lets not add spaces like this. try using paragraph with tabs or a better alternative.